kubeflow/mnist/k8s/mpi_job_mnist.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: "kubeflow.org/v2beta1"
```

Comment:
don't use underscores in kubernetes manifest file names, and put the kind in the end,ie: `mnist-mpijob.yaml`